src/animationComponents/EntityScopes.js

```diff
@@ -527,6 +534,19 @@ const CompoundEntity = React.memo(React.forwardRef(({ id, index, indexArray=[],
           colorChanged = true;
         }
       }
+
```

Comment:
This is a good to have the boundary calculated. We somehow need to smooth the boundary so it looks like a blob. There is quite a lot of info on the web around Metaballs e.g. https://threejs.org/examples/webgl_marchingcubes.html Obviously we are dealing with 2D but maybe there are ideas about how to achieve a blob appearance (I am interested in the boundary - we do no need to worry about material and lighting). Another approach which might work is https://github.com/stevinz/three-subdivide